plugins/source/gcp/resources/plugin/client.go

```diff
@@ -69,6 +72,19 @@ func (c *Client) Sync(ctx context.Context, options plugin.SyncOptions, res chan<
 	if err != nil {
 		return err
 	}
+	var skippedOrgTables []string
+	for _, table := range tables {
+		if c.syncClient.Spec.SkipOrganizationResources && table.Multiplex != nil {
+			// get the name of the multiplexer via reflection
+			multiplexerName := runtime.FuncForPC(reflect.ValueOf(table.Multiplex).Pointer()).Name()
+			if strings.HasPrefix(multiplexerName, "github.com/cloudquery/cloudquery/plugins/source/gcp/client.OrgMultiplex") {
```

Comment:
Might be confusing to see this code in the private version of the repo, maybe only check the function name?